templates.dist/form/ownerform.php

```diff
@@ -58,9 +63,14 @@
 		if ( in_array( $input, array( 'message' )) ) {
 			$isRequiredMessage = $pForm->isRequiredField( 'message' );
 			$additionMessage = $isRequiredMessage ? '*' : '';
-	
-			$line = $pForm->getFieldLabel( 'message' ).$additionMessage.':<br>';
-			$line .= '<textarea name="message">'.$pForm->getFieldValue('message').'</textarea><br>';
+			$isHiddenField = $pForm->isHiddenField('message');
+			if (!$isHiddenField) {
+				$line = $pForm->getFieldLabel( 'message' );
+				$line .= $additionMessage . ':<br>';
+				$line .= '<textarea name="message">' . $pForm->getFieldValue('message') . '</textarea><br>';
+			} else {
+				$line = '<textarea name="message" class="oo-hidden-field" disabled">'.$pForm->getFieldValue('message').'</textarea><br>';
```

Comment:
There is a stray `"` double-quotation after `disabled` and please use `readonly` instead of `disabled`.